moveit_core/robot_trajectory/src/robot_trajectory.cpp

```diff
@@ -101,11 +102,6 @@ void RobotTrajectory::append(const RobotTrajectory& source, double dt, size_t st
 void RobotTrajectory::reverse()
 {
   std::reverse(waypoints_.begin(), waypoints_.end());
-  for (robot_state::RobotStatePtr& waypoint : waypoints_)
```

Comment:
Why are we removing this?

Reply:
@anasarrak I don't quite recall having removed this on purpose. Does something come to mind for you? @mlautman maybe this is the result of syncing with moveit1?

Reply:
@davetcoleman, not sure where this is coming from but [reverting this](https://github.com/AcutronicRobotics/moveit2/commit/000343fee377e777d87fa8480ac2945a5c9e89a3) as you pointed causes yet more problems when compiling:

```bash
/usr/local/include/octomap/OccupancyOcTreeBase.hxx:116:108: warning: unused parameter 'maxrange' [-Wunused-parameter]
  void OccupancyOcTreeBase<NODE>::insertPointCloudRays(const Pointcloud& pc, const point3d& origin, double maxrange, bool lazy_eval) {
                                                                                                           ^
/Users/victor/ros2_moveit_ws/src/moveit2/moveit_core/robot_trajectory/src/robot_trajectory.cpp:108:15: error: no member named 'invertVelocity' in 'moveit::core::RobotState'
    waypoint->invertVelocity();
    ~~~~~~~~  ^
```

I'd suggest we finalize the ROS 2 porting and then we rebase things with respect MoveIt! (1).

Reply:
How shall we proceed about this?

Reply:
The issue here is that `invertVelocity()` has not yet been synced to the PRs base branch but the current master branch has this function (https://github.com/ros-planning/moveit2/pull/59/files#diff-15d4dc0e9e1eee143f7576803dfe8ea2L441). Did you try compiling these changes after a rebase to master? Actually it's the same problem as in PR #59 where `invertVelocity()` was accidently removed from the feature branch (https://github.com/ros-planning/moveit2/pull/59/files#diff-15d4dc0e9e1eee143f7576803dfe8ea2L441).